src/electron/window.ts

```diff
@@ -98,24 +98,22 @@ export function createWindow(url?: string) {
     window.webContents.send(ElectronEvent.FULLSCREEN_CHANGE, false);
   });
 
-  window.on('close', (event) => {
+    window.on('close', (event) => {
```

Comment:
Please make sure to pass ESLint pre-commit checks.

Reply:
Done